erpnext/stock/doctype/delivery_note/delivery_note.py

```
@@ -127,7 +127,15 @@ class DeliveryNote(SellingController):
 		shipping_address_name: DF.Link | None
 		shipping_rule: DF.Link | None
 		status: DF.Literal[
-			"", "Draft", "To Bill", "Completed", "Return", "Return Issued", "Cancelled", "Closed"
+			"",
```

Ruff (0.14.13)

Syntax error in forward annotation: Expected an expression

(F722)